model/proto/model.proto

```diff
@@ -158,3 +158,9 @@ message Batch {
       (gogoproto.nullable) = true
     ];
 }
+
+message DependencyLink {
+  string parent = 1;
+  string child = 2;
+  uint64 call_count = 3;
```

Comment:
missing the new `source` field

Reply:
Trying to work out the correct way to pull this in, are you expecting to see it as a bytes field type with a customtype hooking out to DependencyLinkSource?

Reply:
I see what you mean - I think we should change the current code to remove DependencyLinkSource and use a plain string. The idea was that external sources can submit links data, and there's no reason to have an enum in the code representing those sources, it's enough to have just the constant for "jaeger" as the default source type.